setup.cfg

```
@@ -1,8 +1,49 @@
 [metadata]
+name = spdx-tools
+author = Ahmed H. Ismail
+author_email = ahm3d.hisham@gmail.com
+maintainer = Philippe Ombredanne, SPDX group at the Linux Foundation and others
+maintainer_email = pombredanne@gmail.com
+license = Apache-2.0
 license_file = LICENSE
+description = SPDX parser and tools.
+url = https://github.com/spdx/tools-python
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python :: 2.7
```

We are dropping Python 2.
```suggestion
    Programming Language :: Python :: 3
```